iterative/resource_task.go

```diff
@@ -18,6 +18,10 @@ import (
 	"terraform-provider-iterative/task/common"
 )
 
+var (
+	logTpl = "%s may take several minutes (consider increasing `timeout` https://registry.terraform.io/providers/iterative/iterative/latest/docs/resources/task#timeout). Please wait."
```

Comment:
Asking myself: if we suggest users to increase timeouts indiscriminately, why not do it ourselves and alter the defaults? 🤔 

Operation times are bound to data synchronization, though. It may be better to warn users so they can set the limits that better suit their data. 🙃

Reply:
> if we suggest users to increase timeouts indiscriminately

I just wanted to provide a handy link of how the operation can be extended so they can actually see that they can alter it and its operation its not inevitably broken.

> Operation times are bound to data synchronization,

That true however destroy has also timeout for me just syncing two small files.

Reply:
> _That true however destroy has also timeout for me just syncing two small files_

#486

Reply:
> _I just wanted to provide a handy link of how the operation can be extended so they can actually see that they can alter it and its operation its not inevitably broken._

👍🏼